pkg/vault/test/foundry/BatchRouterERC4626Pool.t.sol

```diff
@@ -758,10 +689,9 @@ contract BatchRouterERC4626PoolTest is BaseERC4626BufferTest {
             );
 
             // The pool gains the wrapped tokens from the buffer and mints BPT to the user.
-            assertApproxEqAbs(
+            assertEq(
                 poolBalances[ybUsdcIdx],
-                waUSDC.convertToShares(erc4626PoolInitialAmount) + vars.wrappedUsdcPoolDelta,
-                MAX_ERROR,
```

Comment:
MAX_ERROR is still defined, and there are two places where it's still used. It can be removed, and those checks changed to `assertEq` (I tried it).

Reply:
Nice catch!